### PR TITLE
Publish only changed settings or modes in MQTT callback

### PR DESCRIPTION
**Pull request by tomrosenback**
Solves issue #33

### CODE REVIEW

**Comment by tomrosenback** [comment]
@Jalle19 Is this ok?

File: app/mqtt.mjs [hunk 1]
```diff
@@ -56,6 +43,43 @@ export const publishValues = async (modbusClient, mqttClient) => {
...
+    await publishTopics(mqttClient, topicMap)
+}
+
+const publishSettings = async (modbusClient, mqttClient, settingName = '') => {
+    // Create a map from topic name to value that should be published
+    let topicMap = {}
+    const settings = await getSettings(modbusClient)
+
+    for (const [setting, value] of Object.entries(settings)) {
+        if(settingName === '' || settingName == setting) {
```
Comment (Jalle19) [inline app/mqtt.mjs:73]: I'm not sure this check is worth it - we're requesting all settings from Modbus anyway so this won't make anything faster AFAICT? Might as well keep it simple.

**Review by Jalle19 — COMMENTED** [review]
Just one minor comment

File: app/mqtt.mjs (hunk 1 above)
Comment (tomrosenback) [reply app/mqtt.mjs:73]: True, I´ll change

File: app/mqtt.mjs [hunk 2]
```diff
@@ -140,14 +161,14 @@ export const configureMqttDiscovery = async (modbusClient, mqttClient) => {
...
         'cascadeP': createSensorConfiguration(configurationBase, 'cascadeP', 'Cascade P-value'),
         'cascadeI': createSensorConfiguration(configurationBase, 'cascadeI', 'Cascade I-value'),
-        'overPressureTimeLeft': createSensorConfiguration(configurationBase, 'overPressureTimeLeft', 'Overpressure time left', {'unit_of_measurement': 'minutes'}),
-        'ventilationLevelTarget': createSensorConfiguration(configurationBase, 'ventilationLevelTarget', 'Ventilation level (target)', {'unit_of_measurement': '%'}),
-        'ventilationLevelActual': createSensorConfiguration(configurationBase, 'ventilationLevelActual', 'Ventilation level (actual)', {'unit_of_measurement': '%'}),
+        'overPressureTimeLeft': createSensorConfiguration(configurationBase, 'overPressureTimeLeft', 'Overpressure time left', { 'unit_of_measurement': 'minutes' }),
+        'ventilationLevelTarget': createSensorConfiguration(configurationBase, 'ventilationLevelTarget', 'Ventilation level (target)', { 'unit_of_measurement': '%' }),
+        'ventilationLevelActual': createSensorConfiguration(configurationBase, 'ventilationLevelActual', 'Ventilation level (actual)', { 'unit_of_measurement': '%' }),
```
Comment (Jalle19) [inline app/mqtt.mjs:171]: There are some unrelated whitespace changes here, can you undo them? I should add a formatter to the project.

Comment (Jalle19) [reply app/mqtt.mjs:171]: Actually, let them be. We'll format the whole thing later with `prettier`.

**Review by Jalle19 — APPROVED** [review]

File: app/mqtt.mjs (hunk 2 above)
Comment (tomrosenback) [reply app/mqtt.mjs:171]: The whole thing is formatted with prettier, as all the other documents touched here.
